dhis-2/dhis-services/dhis-service-data-exchange/src/main/java/org/hisp/dhis/dataexchange/aggregate/AggregateDataExchangeService.java

```diff
@@ -270,8 +270,8 @@ ImportOptions toImportOptions(AggregateDataExchange exchange) {
     if (notNull(request.getImportStrategy())) {
       options.setImportStrategy(request.getImportStrategy());
     }
-    if (notNull(request.getSkipAudit())) {
-      options.setSkipAudit(request.getSkipAudit());
+    if (notNull(request.isSkipAuditOrDefault())) {
+      options.setSkipAudit(request.isSkipAuditOrDefault());
     }
     if (notNull(request.getDryRun())) {
```

Comment:
I know the if check above is not necessary but just makes the code more consistent and pretty.

Reply:
hm... I think more that it is misleading because this will always set the property in the target object, having an `if` just gives the wrong idea about this behaviour.

Reply:
Fine, changed it now.